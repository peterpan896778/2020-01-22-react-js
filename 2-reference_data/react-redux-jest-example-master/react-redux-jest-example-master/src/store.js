import { createStore } from 'redux';
import { composeWithDevTools } from 'redux-devtools-extension';
import reducer from 'src/reducers';

const store = createStore(
  reducer,
  composeWithDevTools(),
);

export default store;
