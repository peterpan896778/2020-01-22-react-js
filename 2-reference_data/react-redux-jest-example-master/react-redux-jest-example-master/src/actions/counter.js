import { INCREMENT, DECREMENT } from 'src/constants';

export const increment = () => ({
  type: INCREMENT,
});

export const decrement = () => ({
  type: DECREMENT,
});
