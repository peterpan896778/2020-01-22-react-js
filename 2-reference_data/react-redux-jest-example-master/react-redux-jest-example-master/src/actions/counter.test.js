import store from 'src/store';
import { increment, decrement } from './counter';

describe('Counter actions', () => {
  test('Counter increment action', () => {
    expect(store.dispatch(increment())).toMatchSnapshot();
  });

  test('Counter decrement action', () => {
    expect(store.dispatch(decrement())).toMatchSnapshot();
  });
});
