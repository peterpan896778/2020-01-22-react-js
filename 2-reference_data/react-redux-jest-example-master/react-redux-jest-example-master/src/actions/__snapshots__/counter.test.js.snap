// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter actions Counter decrement action 1`] = `
Object {
  "type": "/App/DECREMENT",
}
`;

exports[`Counter actions Counter increment action 1`] = `
Object {
  "type": "/App/INCREMENT",
}
`;
